docs/build-linux.md

```diff
@@ -1,6 +1,185 @@
+  not executables. They are, too, taken from the FreeDOS project; the binaries
+  were created from the source `*.KEY` text files, using a specialized tools,
+  written in Pascal - they are, too, part of the FreeDOS project, search for
+  `KEYB200S.ZIP`, `KEYB200X.ZIP`, `KC200S.ZIP`, and `KC200X.ZIP` files.
+  Probably all the other DOSBox forks uses such files in some form; it's just
+  most of them store the files content as an array of binary data, like the
+  [original DOSBox does](https://sourceforge.net/p/dosbox/code-0/HEAD/tree/dosbox/tags/RELEASE_0_74_3/src/dos/dos_keyboard_layout_data.h).
+  Running DOSBox Staging without these files is completely unsupported; even if
+  it seems to work for you, the internationalization features will malfunction.
+
+## Building using system libraries
+
+These are generic, distro-independent building instructions.
 
-## Minimum build requirements
+1. Install the necessary build tools:
+
+- GCC or Clang compiler (the compiler has to support C++20)
+- GIT
+- CMake
+- pkg-config
+
+2. Install the dependencies, development packages are needed too:
+
+- SDL 2.x
+- SDL2_net
+- IIR
+- OpusFile
+- MT32Emu
+- FluidSynth
+- ALSA
+- libpng
+- OpenGL headers
+
+3. Clone DOSBox Staging:
+
+```bash
+git clone https://github.com/dosbox-staging/dosbox-staging.git
+```
+
+4. Configure the sources and build DOSBox Staging:
+
+```bash
+cd dosbox-staging
+cmake --preset=release-linux
+cmake --build --preset=release-linux
+```
+
+5. You can now launch DOSBox with the command:
+
+``` bash
+./build/release-linux/dosbox
+```
+
+## Building using vcpkg
+
+1. Install the necessary build tools:
+
+- for Ubuntu:
+
+```bash
+sudo apt-get install git build-essential pkg-config cmake curl ninja-build \
+             autoconf bison libtool libgl1-mesa-dev libsdl2-dev
+```
+
+2. Install the `vcpkg` tool:
+
+- clone the `vcpkg` tool into your home directory
+
+```bash
+cd ~
+git clone https://github.com/microsoft/vcpkg.git
+```
+
+- bootstrap the `vcpkg` tool:
+
+```bash
+cd vcpkg
+./bootstrap-vcpkg.sh -disableMetrics
+```
+
+- set the vcpkg path in your compile shell (it is recommended to add the command
+  to your shell startup script, usually `~/.bashrc`)
+
+```bash
+export VCPKG_ROOT=$HOME/vcpkg
+```
+
+3. Clone DOSBox Staging into your home directory:
+
+```bash
+cd ~
+git clone https://github.com/dosbox-staging/dosbox-staging.git
+```
+
+4. Configure the sources and build DOSBox Staging:
+
+```bash
+cd dosbox-staging
+cmake --preset=release-linux-vcpkg
+cmake --build --preset=release-linux-vcpkg
+```
+
+5. You can now launch DOSBox with the command:
+
+```bash
+./build/release-linux/dosbox
+```
+
+## Sanitizer build
+
+There are two (mutually exclusive) sanitizer settings available:
+- `OPT_SANITIZER` - detects memory errors and undefined behaviors
+- `OPT_THREAD_SANITIZER` - data race detector
+
+To use any of these, pass the appropriate option when configuring the sources,
```

Comment:
"configuring the project"
"configuring the build system"

I don't think you can configure the sources 😄